app/public/js/app.js

```diff
@@ -24,42 +24,36 @@ $('.documentation table').addClass('table table-striped table-bordered');
 
 var quickSearchData = null;
 
-var $quickSearchTypeahead = $('#quick-search-typeahead>input');
-$quickSearchTypeahead.typeahead({
-    source: function (query, process) {
-        if (quickSearchData) {
-            process(quickSearchData.names);
-        } else {
-            var url = $quickSearchTypeahead.attr('data-action');
-
-            $.get(url, {}, function (data) {
-                quickSearchData = JSON.parse(data);
-                process(quickSearchData.names);
+var $quickSearchTypeahead = $('#quick-search-typeahead').find('>input');
+    $quickSearchTypeahead.on("focus", function () {
+        var url = $quickSearchTypeahead.attr('data-action');
+        $.get(url, {}, function (quickSearchData) {
+            $quickSearchTypeahead.typeahead({
+                source: quickSearchData.names,
+                items: 'all',
+                updater: function (item) {
+                    // get text to display in quick search box
+                    var displayText = item;
+                    console.log(item);
```

Comment:
probably want to remove the `console.log`

Reply:
You are right, I removed it in https://github.com/piwik/developer-documentation/pull/200/commits/a163bae1e3379a5d0feda1b3dbee366aea6bb542 and forgot to also do so here.

But keep in mind that this function doesn't exist any more with the full text search in #200 as I am using a maintained fork of twitter bootstrap there, because it's more user friendly (results can be selected with the arrow keys)